modules/verifying-serverless-app-deployment.adoc

```diff
@@ -30,55 +30,55 @@ $ oc get ksvc <service_name>
 .Example output
 [source,terminal]
 ----
-NAME            URL                                        LATESTCREATED         LATESTREADY           READY   REASON
-event-delivery   http://event-delivery-default.example.com   event-delivery-4wsd2   event-delivery-4wsd2   True
+NAME       URL                                   LATESTCREATED    LATESTREADY      READY   REASON
+showcase   http://showcase-default.example.com   showcase-00001   showcase-00001   True
```

Comment:
Do you have a pointer of the draft doc for this? Is the output provided on jira?